rootfs/strap.py

```
@@ -51,12 +52,16 @@
     install_to = parts[1]
 
     if package == "remove":
-        package = parts[1]
+        package = re.sub(r'[^a-zA-Z0-9._+\-]', '', os.path.basename(parts[1]))
+        if not package:
+            print("  ==> Error: invalid package name")
+            exit(1)
```

Ruff (0.15.17)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)